akka-projection-jdbc/src/test/scala/akka/projection/jdbc/JdbcProjectionSpec.scala

```diff
@@ -895,7 +876,7 @@ class JdbcProjectionSpec
           .exactlyOnce(
             projectionId,
             sourceProvider = sourceProvider(system, entityId),
-            jdbcSessionCreator,
+            jdbcSessionFactory,
             handler = handler)
           .withRestartBackoff(1.second, 2.seconds, 0.0, maxRestarts = 0)
 
```

Comment:
good, you should also add a `JdbcProjectionTest` corresponding to CassandraProjectionTest for brief testing of the javadsl

Reply:
I was planing to add it on another PR